packages/forma-36-react-components/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -1,53 +1,62 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, boolean } from '@storybook/addon-knobs';
-import { action } from '@storybook/addon-actions';
 
-import Checkbox from './Checkbox';
+import Checkbox, { CheckboxProps } from './Checkbox';
 import Flex from '../Flex/Flex';
 import SectionHeading from '../Typography/SectionHeading';
 
-storiesOf('Components/Checkbox', module)
-  .addParameters({
-    propTypes: Checkbox['__docgenInfo'],
-    component: Checkbox,
-  })
-  .add('default', () => (
+export default {
+  title: 'Components/Checkbox',
+  component: Checkbox,
+  parameters: {
+    propTypes: [Checkbox['__docgenInfo']],
+  },
+  argTypes: {
+    className: { control: { disable: true } },
+    testId: { control: { disable: true } },
+  },
+};
+
+export const basic = (args: CheckboxProps) => <Checkbox {...args} />;
+
+basic.args = {
+  id: 'Checkbox',
+  labelText: 'some label text',
```

Comment:
I think this is not a prop of Checkbox, is it?

Reply:
@gui-santos ups, totally right.thx! removed.